common/persistence/sql/sqlplugin/history_execution.go

```diff
@@ -66,6 +67,7 @@ type (
 		WorkflowID       string
 		RunID            primitives.UUID
 		CreateRequestID  string
+		StartTime        *time.Time
```

Comment:
looks like other definitions like `TimerTasksRow` uses `time.Time`.

Reply:
Are those fields also nullable though?

Reply:
oh you're right. I missed that. time task should also have a timestamp.